src/utils/lexerUtils.ts

```diff
@@ -7,6 +7,7 @@ export enum XmlTagName {
     ArtifactId = "artifactId",
     Version = "version",
     Dependencies = "dependencies",
+    Exclusions = "exclusions",

```

Comment:
In this PR, what's it for?

Reply:
For inserting exclusion node in the pom file